dashboard/test/controllers/scripts_controller_test.rb

```diff
@@ -934,44 +901,6 @@ class ScriptsControllerTest < ActionController::TestCase
     assert unit.is_maker_unit
   end
 
-  test 'set_and_unset_teacher_resources' do
-    sign_in create(:levelbuilder)
-    Rails.application.config.stubs(:levelbuilder_mode).returns true
-
-    unit = create :script
-    stub_file_writes(unit.name)
-
-    # Test doing this twice because teacher_resources in particular is set via its own code path in update_teacher_resources,
-    # which can cause incorrect behavior if it is removed during the Script.add_unit while being added via the
-    # update_teacher_resources during the same call to Script.update_text
```

Comment:
@bethanyaconnor any chance you can give me a clue on whether you think an equivalent test will be necessary for migrated teacher resources when I remove this?

Reply:
I'm not 100% sure what this comment is saying but a quick look makes me think we *do not* need an equivalent test. In particular, because `teacher_resources` used to be stored on the script itself, `update_text` would update the script (without updating `teacher_resources` _then_ update `teacher_resources` in a second step. With migrated teacher resources, the script object is not actually updated in `update_migrated_teacher_resources` so I don't imagine the same problem would occur?

On a broader level, we do have tests covering adding migrated teacher resources, though I'm not sure we have tests covering removing them (like this test does at the end), so that might be a nice add but definitely not needed in this PR